src/murmur/murmur.pro

```diff
@@ -88,8 +88,23 @@ ice {
 	win32:CONFIG(debug, debug|release) {
 		LIBS *= -lIceD -lIceUtilD
 	} else {
-		LIBS *= -lIce -lIceUtil
+		# check Ice version, 3.7 merged IceUtil into Ice
+		ICE_VERSION = $$system(slice2cpp --version 2>&1)
+		ICE_MAJOR_VERSION = $$section(ICE_VERSION, ., 0, 0))
```

Comment:
The error is because you have an extra right parenthesis at the end here.
So ICE_MAJOR_VERSION gets set to `3)` :cry: